js/components/uswds_checkbox_widget.jsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
-const USWDSCheckboxWidget = function(props) {
+function USWDSCheckboxWidget({ label, value }) {
   return (
     <div>
       <input 
         type="checkbox" 
         id={props.label} 
         name={props.label} 
         className={props.value ? "checked" : "unchecked"} 
-        onClick={() => props.onChange(!props.value)} 
+        onChange={() => props.onChange(!props.value)} 
```

Comment:
Oops, this is on me, but since we removed `props` as arguments, these will all be undefined or an error. We should just add `props` back as an argument instead of the object destructoring.